pkg/cloudprovider/providers/aws/aws_routes.go

```diff
@@ -49,6 +49,12 @@ func (s *AWSCloud) findRouteTable(clusterName string) (*ec2.RouteTable, error) {
 
 // ListRoutes implements Routes.ListRoutes
 // List all routes that match the filter
+func getInstanceAndCIDR(r *ec2.Route)(string, string) {
```

Comment:
Personally yes I think it'll be clearer if we just inline it in the two spots.  I don't think we need to worry about skipping where `destinationCIDR == ""` in the first loop, because I don't think that will actually happen much in practice (and worst case we just query a few more instances).  So then the repetition isn't too bad.